Library/Homebrew/blacklist.rb

```diff
@@ -1,16 +1,5 @@
 def blacklisted? name
   case name.downcase
-  when 'screen', /^rubygems?$/ then <<-EOS.undent
```

Comment:
In that case, we can just edit the blacklist message. Append the information that you can install from tap if you need a newer version of it.